runbooks/runbook.md

```diff
@@ -0,0 +1,112 @@
+# UPP - Annotations Mapper
+
+Annotations Mapper reads all QMI native metadata messages from the `NativeCmsMetadataPublicationEvents` queue and maps them to a JSON message, which is forwarded to the `ConceptAnnotations` queue.
+
+## Code
+
+annotations-mapper
+
+## Primary URL
+
+<https://github.com/Financial-Times/annotations-mapper>
+
+## Service Tier
+
+Platinum
+
+## Lifecycle Stage
+
+Production
+
+## Delivered By
+
+content
+
+## Supported By
+
+content
+
+## Known About By
+
+- dimitar.terziev
+- elitsa.pavlova
+- kalin.arsov
+- ivan.nikolov
+- miroslav.gatsanoga
+- marina.chompalova
+- hristo.georgiev
+- donislav.belev
+- mihail.mihaylov
+- boyko.boykov
+
+## Host Platform
+
+AWS
+
+## Architecture
+
+None provided
+
+## Contains Personal Data
+
+No
+
+## Contains Sensitive Data
+
+No
+
+## Failover Architecture Type
+
+ActiveActive
+
+## Failover Process Type
+
+NotApplicable
```

Comment:
This is not the correct value. And for the following fields too.

Reply:
I referred to this repo: https://github.com/Financial-Times/mongo-hot-backup/pull/22/files, probably we'll need to fix it as well.

Reply:
This service runs only in the EU cluster and therefore failover is not applicable for it.